templates/ts/copilot-plugin-from-scratch/appPackage/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.15",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
+    "developer": {
+        "name": "Teams App, Inc.",
+        "websiteUrl": "https://www.example.com",
+        "privacyUrl": "https://www.example.com/privacy",
+        "termsOfUseUrl": "https://www.example.com/termsofuse"
+    },
+    "icons": {
+        "color": "color.png",
+        "outline": "outline.png"
+    },
+    "name": {
+        "short": "copilot-plugin-tpl-${{TEAMSFX_ENV}}",
+        "full": "Full name for copilot-plugin-tpl"
+    },
+    "description": {
+        "short": "Short description of copilot-plugin-tpl",
+        "full": "Full description of copilot-plugin-tpl"
+    },
+    "accentColor": "#FFFFFF",
+    "bots": [],
+    "composeExtensions": [],    
```

Comment:
Should sme related settings be added here?

Reply:
Yeah, we might make some changes to this later on, the SME manifest configuration hasn't been finalized yet.